docs/providers/aws/events/event-bridge.md

```diff
@@ -186,3 +186,38 @@ functions:
               eventTime: '$.time'
             inputTemplate: '{"time": <eventTime>, "key1": "value1"}'
 ```
+
```

Comment:
It should be documented that support for `deadLetterConfig` and `retryPolicy` is only available when it is using native CloudFormation